docs/UserGuide.md

```diff
@@ -447,13 +448,13 @@ Format: `delete <index>`
 * The index **must be a positive integer** 1, 2, 3, …
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
+* `list` followed by `delete 2` deletes the 2nd person in PayMeLah.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all entries: `clear`
+### Clearing all people from PayMeLah: `clear`
```

Comment:
Do you think "every person" be better than "all people"?

Reply:
I think "all people" sounds better; do you want me to make this change anyway?